README.md

```diff
@@ -7,4 +7,13 @@
 ## Instructions
 Everyone loves a good gif of puppies and kids doing funny things, so let's use jQuery to make sure we have a good one on our page.
 
-We'll be coding our solution in 
\ No newline at end of file
+We'll be coding our solution in `html/index.html`.
+
++ Link jQuery to `index.html` using the following link `http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js`.
+
++ Before the closing body tag, use jQuery to append this gif (https://s3.amazonaws.com/learn-verified/painting-with-dog.gif) to the div with the ID `image`. 
```

Comment:
should we include a demo of appending an html element vs just plain text? This will be more fun if they add the img tag vs just adding the link